usaspending_api/disaster/v2/views/agency/count.py

```diff
@@ -40,7 +29,29 @@ def post(self, request: Request) -> Response:
                         "submission_fiscal_month"
                     ),
                 )
-            ),
+            )
+
+        sub_queries.append(
+            Q(
+                submission__quarter_format_flag=True,
+                submission__reporting_fiscal_year__lte=self.last_closed_quarterly_submission_dates.get(
+                    "submission_fiscal_year"
+                ),
+                submission__reporting_fiscal_quarter__lte=self.last_closed_quarterly_submission_dates.get(
+                    "submission_fiscal_quarter"
+                ),
+            )
+        )
+
+        sub_queryset = sub_queries.pop()
+        for query in sub_queries:
+            sub_queryset |= query
```

Comment:
You take the last element, then or it with all elements? Is this just a workaround to get a list to a Q?

Reply:
Workaround to get it to a Q with the possibility of no monthly submission query. Ideally, this can all be removed once we get into August and there is a closed submission window to return for monthly.